src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendCol = document.querySelector('.append-column');
+const removeCol = document.querySelector('.remove-column');
+
+appendRow.addEventListener('click', (e) => {
+  const listTr = document.querySelectorAll('tr');
+
+  if (listTr.length < 10) {
+    const tr = listTr[0].outerHTML;
+
+    listTr[0].insertAdjacentHTML('beforebegin', tr);
+    removeRow.disabled = false;
+    e.target.disabled = listTr.length === 9;
```

Comment:
why do you need these 2 lines?
```suggestion
    removeRow.disabled = listTr.length === 9;
```

Reply:
same for 3 other cases